scrapi/harvesters/neurovault.py

```diff
@@ -0,0 +1,79 @@
+"""
+NeuroVault (neurovault.org) harvester of public collections for the
+SHARE Notification Service.
+
+Example API query: http://neurovault.org/api/collections/?format=json
+"""
+
+from __future__ import unicode_literals
+
+import json
+import logging
+# from datetime import date, timedelta, datetime
+import re
+from scrapi import requests
+# from scrapi import settings
+from scrapi.base import JSONHarvester
+from scrapi.linter.document import RawDocument
+from scrapi.base.helpers import (build_properties, datetime_formatter,
+                                 default_name_parser)
+
+logger = logging.getLogger(__name__)
+
+
+def process_contributors(authors):
+
+    if authors is None:
+        return []
+    authors = re.split(',\s|\sand\s', authors)
+    return default_name_parser([a for a in authors])
+
+
+class NeuroVaultHarvester(JSONHarvester):
+    short_name = 'neurovault'
+    long_name = 'NeuroVault.org'
+    url = 'http://www.neurovault.org/'
+    count = 0
+
+    @property
+    def schema(self):
+        return {
+            'contributors': ('/authors', process_contributors),
+            'uris': {
+                'canonicalUri': '/url',
```

Comment:
Yep, no problem! [This is the most up to date schema](https://osf.io/wur56/wiki/Schema/?view), and [this is the repository for it](https://github.com/CenterForOpenScience/SHARE-schema). It is in JSON-schema (and the one in the repo is YAML, because it is easier to write by hand). If you have any questions let us know. Also feel free to propose changes in the SHARE-schema repo.